File: 3.0/python/8.py
minx, miny = 10000000000, 10000000000
maxx, maxy = 0, 0

for i in range(int(input())):
    x, y = map(int, input().split())
    if x < minx: minx = x
    if y < miny: miny = y
    if x > maxx: maxx = x
    if y > maxy: maxy = y

print(minx, miny, maxx, maxy)

File: 3.0/golang/8.go
package main

import "fmt"

func main() {
	minx, miny := 10000000000, 10000000000
	maxx, maxy := 0, 0

	var k int
	fmt.Scanln(&k)

	for i := 0; i < k; i++ {
		var x, y int
		fmt.Scanf("%d %d\n", &x, &y)

		if x < minx {
			minx = x
		}
		if y < miny {
			miny = y
		}
		if x > maxx {
			maxx = x
		}
		if y > maxy {
			maxy = y
		}
	}

	fmt.Println(minx, miny, maxx, maxy)
}
